_documentation/user/flags/blindness.md

```diff
@@ -5,4 +5,19 @@ type: Bad
 summary: Can't see out window. Radar still works.
 ---
 
-...
+The Blindness flag blocks sight of the map, tanks and shots from outside window viewing perspective.
```

Comment:
Reads cleaner, but perhaps the following would read slightly better:

> The Blindness (B) flag blocks sight of the map, tanks, and shots from the heads-up display (HUD).

As flags effects are per tank, I assume this would work fine.